corral/cmd/cmd_fetch.pony

```diff
@@ -9,10 +9,8 @@ class CmdFetch
 
   new create(ctx': Context, cmd: Command) =>
     ctx = ctx'
-    //ctx.log.info("fetch: " + cmd.string())
```

Comment:
Another commented-out log to restore please.

Reply:
It's already restored, github is not assuming that in this conversation :+1:

Reply:
Ah, I missed that, sorry. All good.